src/main/kotlin/org/jitsi/rtp/rtcp/rtcpfb/payload_specific_fb/RtcpFbRembPacket.kt

```diff
@@ -92,8 +91,16 @@ class RtcpFbRembPacket(
             buf.getBitsAsInt(baseOffset + BR_OFF, 0, 6)
         fun getBrMantissa(buf: ByteArray, baseOffset: Int): Int =
                 (buf.getBitsAsInt(baseOffset + BR_OFF, 6, 2) shl 16) + buf.getShortAsInt(baseOffset + BR_OFF + 1)
-        fun getBitrate(buf: ByteArray, baseOffset: Int): Long =
-                (getBrMantissa(buf, baseOffset) * 2.0.pow(getBrExp(buf, baseOffset).toDouble())).toLong()
+        fun getBitrate(buf: ByteArray, baseOffset: Int): Long {
+            val mantissa = getBrMantissa(buf, baseOffset)
+            val exp = getBrExp(buf, baseOffset)
+            val brBps = mantissa.toLong() shl exp
+            if ((brBps shr exp).toInt() != mantissa) {
+                return -1
+            }
```

Comment:
So if we get a REMB with a bitrate greater than Long.MAX_VALUE then getBitrate will return -1?  Is that what we want?

Reply:
Here I am trying to detect the overflow situation given the specific limits of our system. I think that for all practical purposes this is correct, i.e. if we ever get a REMB with an apparent bitrate that is greater than Long.MAX_VALUE, I'd suspect an error somewhere in our code and not a network that has such big apparent bandwidth. As an additional step, we could log a warning. As an alternative, we could throw an exception, or return Long.MAX_VALUE.

Reply:
The REMB we receive is coming from the other party, not ourselves.  I agree that for our system it would be broken if we ever got a REMB that big, but I could vaguely imagine someone trying to send the maximum value to mean something like "unbounded."

I'd argue in favor of returning Long.MAX_VALUE.